commands/createfixpullrequests_test.go

```diff
@@ -0,0 +1,17 @@
+package commands
+
+import (
+	"github.com/sassoftware/go-rpmutils"
+	"testing"
+)
+
+func TestCreateFixPullRequests(t *testing.T) {
+	testCreateFixPullRequests(t, "", "go-proj")
+}
+
+func testCreateFixPullRequests(t *testing.T, workingDirectory, projectName string) {
```

Comment:
Tests should be implemented. I suggest we add them in a follow-up PR, to allow @asafgabai create his PR in the meantime.